plugins/phpmyadmin-rule-exclusions-before.conf

```diff
@@ -187,20 +223,28 @@ SecRule REQUEST_FILENAME "@endsWith /import.php" \
     ctl:ruleRemoveTargetById=942140;ARGS:db,\
     ctl:ruleRemoveTargetByTag=attack-sqli;ARGS:prev_sql_query,\
     ctl:ruleRemoveTargetById=932130;ARGS:prev_sql_query,\
+    ctl:ruleRemoveTargetById=933210;ARGS:prev_sql_query,\
     ctl:ruleRemoveTargetById=942110;ARGS:sql_delimiter,\
     ctl:ruleRemoveTargetByTag=attack-sqli;ARGS:sql_query,\
+    ctl:ruleRemoveTargetByTag=attack-xss;ARGS:sql_query,\
```

Comment:
I am not an expert, but curious about why are we removing XSS rules from a SQL query parameter? Did XSS cause any problems already?

Reply:
`sql_query` parameter holds the whole SQL command, including the data which can contain anything, for example HTML code (which is triggering XSS rules).

Reply:
Got it. Thanks!